spec/test_app/app/controllers/basechurch/v1/announcements_controller_spec.rb

```diff
@@ -126,7 +127,12 @@
 
   describe 'GET /announcements/:id' do
     let(:announcement) do
-      create(:bulletin_with_announcements, announcements_count: 1).announcements.first
+      a = create(:bulletin_with_announcements, announcements_count: 1).
+        announcements.
+        first
+      a.url = 'http://nba.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.